### PR TITLE
Show additional plugin headers values in `wp plugin get <plugin>` output

### PR DESCRIPTION
**Pull request by ernilambar**
* Plugin header `RequiresPlugins` is also displayed if exists.
* Added feature test for plugin get. One scenario for `RequiresPlugins` and one with basic test in separate file `plugin-get.feature`

Related: https://github.com/wp-cli/extension-command/issues/407

### CODE REVIEW

File: src/Plugin_Command.php [hunk 1]
```diff
@@ -961,6 +961,10 @@ public function get( $args, $assoc_args ) {
 			'status'      => $this->get_status( $file ),
 		];
 
+		if ( isset( $plugin_data['RequiresPlugins'] ) && ! empty( $plugin_data['RequiresPlugins'] ) ) {
```
Comment (swissspidy) [inline src/Plugin_Command.php:964]: Should we also add the field if it's empty?

**Comment by swissspidy** [comment]
Makes we wonder whether we shouldn't add more information from `get_plugin_data` to this list by default. For example the `RequiresWP` and `RequiresPHP` information could be useful (both added in 5.3), or `UpdateURI` (added in 5.8)

**Comment by ernilambar** [comment]
@swissspidy We could keep `name`, `title`, `author`, `version`, `description` and `status` as default fields and other as optional `AVAILABLE FIELDS`. And for available optional fields we could add those new fields, like `requires_plugins`, `requires_php`, etc.

File: src/Plugin_Command.php [hunk 2]
```diff
@@ -941,29 +941,70 @@ public function install( $args, $assoc_args ) {
...
+			'requires_wp'      => 'RequiresWP',
+			'requires_php'     => 'RequiresPHP',
+			'requires_plugins' => 'RequiresPlugins',
+		];
+
+		foreach ( $require_fields as $field_key => $data_key ) {
+			if ( isset( $plugin_data[ $data_key ] ) && ! empty( $plugin_data[ $data_key ] ) ) {
+				$plugin_obj->{$field_key} = $plugin_data[ $data_key ];
+			}
+		}
```
Comment (swissspidy) [inline src/Plugin_Command.php:1004]: Can we simplify this?

```suggestion
			'name'             => Utils\get_plugin_name( $file ),
			'title'            => $plugin_data['Name'],
			'author'           => $plugin_data['Author'],
			'version'          => $plugin_data['Version'],
			'description'      => wordwrap( $plugin_data['Description'] ),
			'status'           => $this->get_status( $file ),
			'requires_wp'      =>  ! empty( $plugin_data['requires_wp'] ) ? $plugin_data['requires_wp'] : '',
			'requires_php'     => ! empty( $plugin_data['requires_php'] ) ? $plugin_data['requires_php'] : '',
			'requires_plugins' => ! empty( $plugin_data['requires_plugins'] ) ? $plugin_data['requires_plugins'] : '',
		];
```

**Comment by swissspidy** [comment]
Now that #417 is merged, merging `main` into this branch should fix the tests

**Review by swissspidy — APPROVED** [review]

**Pull request merged**